nomad/resource_acl_policy.go

```diff
@@ -43,10 +44,78 @@ func resourceACLPolicy() *schema.Resource {
 				Required:    true,
 				Type:        schema.TypeString,
 			},
+
+			"workload_identity": {
```

Comment:
```suggestion
			"job_acl": {
```

It's better to keep the field name consistent with the API. The variables and function names below should be updated as well.